release/rpm/README.md

```
@@ -0,0 +1,35 @@
+./build-rpm-repo.sh
+```
+
+This will produce the RPM binaries under `out/` and index them with `createrepo_c`.
+
+## COPR Build & Distribution
+
+To build and host packages via Fedora COPR:
+1. Create a project in [Fedora COPR](https://copr.fedorainfracloud.org/) named `ryoku`.
+2. Submit builds using `copr-cli build ryoku <specfile-or-srpm>`.
+3. Users can enable the repository using:
+   ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)